### ISSUE TITLE
Hyprland sizing issue

### ISSUE THREAD

**Issue by ViktorTHR** [neovide/neovide#2248]
First off, apologies if this issue should be posted on the Hyprland page.

**Describe the bug**
When fullscreen mode is not set in windorulev2 the window spawns in an unfocused maximized mode, does not follow tiling rules until I toggle through the floating mode for it and has scaling issues in floating mode, displaying a small window with no readability. 

I've tried setting the size for it in hyprland.conf, tried setting it to tiling mode, tried to nofullscreenrequest, which just stops it from opening completely.

Another thing, the launch animation is kinda jittery and always begins in the center of the screen and then jumps to the corner where the tilied window should end up at, so, that's another thing that doesn't follow the general tiling/animation rules. 

**To Reproduce**
Launch Neovide through any means.

**Expected behavior**
The window follows the tiling/scaling rules of all other windows in the WM.

**Screenshots**
![nvim_sc](https://github.com/neovide/neovide/assets/97907551/7421edb1-d165-4bb7-8a0c-b72b20eb9f30)

**Desktop (please complete the following information):**
 Latest Arch
 Latest Hyprland and Neovide from extra repo.

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**
[neovide_rCURRENT.log](https://github.com/neovide/neovide/files/13859892/neovide_rCURRENT.log)

**Additional context**

When I try to run it with the --size command from the terminal it just says error: the argument '--size <SIZE>' cannot be used with '--maximized', I have tried setting maximized=false in the config.toml but that didn't do anything, I have also checked that I don't have --maximized as a launch parameter in the hyprland.conf
Seems to me it's configured to run maximized always and I just can't figure out from where it's getting that.

First time posting on here so sorry if it's not the right way to do it.

**Comment by fredizzimo** [comment]
Check that you don't have the environment variable `NEOVIDE_MAXIMIZED` set somewhere.

**Comment by fredizzimo** [comment]
I should have checked the log. It's the autosaved settings in `.local/share/neovide/neovide-settings.json`. 

Try deleting that file.


We shouldn't save and use those by default.

**Comment by fredizzimo** [comment]
@ViktorTHR, does deleting the file fix the issue?

**Comment by ViktorTHR** [comment]
> @ViktorTHR, does deleting the file fix the issue?

The file is present and has 
{“window”:”Maximized”}

If I delete it a new one is created the next time i opne Neovide.
I’ve tried changing it to jsonc to comment it out but that file is ignored and a new json is created. 
What can I put instead of “Maximized” to get the desired effect?

Thanks for the help!

**Comment by fredizzimo** [comment]
Yes, we really need to fix this properly,  but until then you can set 
https://neovide.dev/configuration.html#remember-previous-window-size to false, and delete the file

There's also a `neovide_remember_window_position` that you can set.

**Comment by ViktorTHR** [comment]
> Yes, we really need to fix this properly, but until then you can set https://neovide.dev/configuration.html#remember-previous-window-size to false, and delete the file
> 
> There's also a `neovide_remember_window_position` that you can set.

Thanks a lot!
It now follows the tiling rules properly and works as it should!

Only "problem" left is the floating window size is still the small box shown in the screenshot, but at least now it's placed where it should be.
